docs/query-builder.txt

```diff
@@ -9,588 +9,1095 @@ Query Builder
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin query type
+   :end-before: end query type
 
-   Car::where('name', 'Toyota')->decrement('weight', 50);
+To learn more about the type codes and query operator, see
+:manual:`$type </reference/operator/query/type/>`
+in the {+server-docs-name+}.
 
-The number of updated objects is returned:
+.. _laravel-query-builder-mod:
 
-.. code-block:: php
+Match a Value Computed with Modulo Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   $count = User::increment('age');
+The following example shows how to pass the ``mod``
+query operator with the ``where()`` query builder
+method to match documents by using the expression
+"``year % 2 == 0``", which matches even values for
+the ``year`` field:
 
-You may also specify more columns to update:
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin query modulo
+   :end-before: end query modulo
 
-.. code-block:: php
-
-   Cat::where('age', 3)
-       ->increment('age', 1, ['group' => 'Kitty Club']);
+To learn more about this query operator, see
+:manual:`$mod </reference/operator/query/mod/>`
+in the {+server-docs-name+}.
 
-   Car::where('weight', 300)
-       ->decrement('weight', 100, ['latest_change' => 'carbon fiber']);
+.. _laravel-query-builder-regex:
 
-MongoDB-specific operators
+Match a Regular Expression
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In addition to the Laravel Eloquent operators, all available MongoDB query
-operators can be used with ``where``:
-
-.. code-block:: php
-
-   User::where($fieldName, $operator, $value)->get();
+The following example shows how to pass the ``REGEX``
+query operator with the ``where()`` query builder
+method to match documents that contain a ``title``
+field that matches the specified regular expression:
 
-It generates the following MongoDB filter:
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin query whereRegex
+   :end-before: end query whereRegex
 
-.. code-block:: ts
+To learn more about regular expression queries in MongoDB, see
+:manual:`$regex </reference/operator/query/regex/>`
+in the {+server-docs-name+}.
 
-   { $fieldName: { $operator: $value } }
+.. _laravel-query-builder-whereRaw:
 
-**Exists**
+Run MongoDB Query API Operations Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Matches documents that have the specified field.
+The following example shows how to use the ``whereRaw()``
+query builder method to run a query operation written by
+using the MongoDB Query API syntax:
 
-.. code-block:: php
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin query raw
+   :end-before: end query raw
 
-   User::where('age', 'exists', true)->get();
+The following code shows the equivalent MongoDB query:
```

Comment:
```suggestion
The following code shows the equivalent MongoDB Shell query:
```

Reply:
Thanks for the suggestion! I think it could be unexpected to introduce MongoDB Shell in passing, so I'll re-iterate that it is MongoDB Query API syntax.